api/Document.json

```
@@ -10812,7 +10812,9 @@
               "version_added": "1.3"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": false,
+              "partial_implementation": true,
+              "notes": "The <code>selectstart</code> event never fires and never invokes this handler."
```

Sorry, I messed up my explanation in the issues. This is my fault, completely.

Since we're setting this as `"version_added": false` we should drop the partial and note on this one. This is a subtlety in how we record data for events: the event itself `selectstart` never fires, so it's `false`, but the handler exists so it's truthy. Sorry for the confusion here.

```suggestion
              "version_added": false
```

---

@ddbeck Thanks a lot again for contributing your time. I am fully aware that you'd have been quicker making the changes yourself and I hope that your investment will yield some return (me engaging more in MDN - have at least a "to do" labelled "read MDN contribution docs" on my list). Updated ...

---

@ddbeck Oh, could I have just clicked on "Commit suggestion" in order to add that to my pull-request? Just learned something new about github, I guess.

---

> you'd have been quicker making the changes yourself

Maybe, but I find I'm much less likely to do something foolish (or at least catch it before committing) if I have to explain it to someone else first. 😄